WellCoffeeInventoryAPI/src/main/java/com/team_lightyear/WellCoffeeInventoryAPI/services/WebApplicationConfig.java

```diff
@@ -0,0 +1,31 @@
+package com.team_lightyear.WellCoffeeInventoryAPI.services;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebApplicationConfig implements WebMvcConfigurer {
+
+//    @Bean
+//    public AuthenticationFilter authenticationFilter() {
```

Comment:
Is the AuthenticationFilter.java doing anything since this section is commented out?  Is that file needed since we're not using Thymeleaf?